Makefile

```diff
@@ -22,7 +22,11 @@ PKGS = $(shell go list ./...)
 PLATFORMS := windows linux darwin
 os = $(word 1, $@)
 
-DOCKER_IMAGE_NAME=go-spacemesh
+ifeq ($(BRANCH),develop)
+        DOCKER_IMAGE_NAME := go-spacemesh
```

Comment:
rename DOCKER_IMAGE_NAME to DOCKER_IMAGE_REPO

Reply:
Done